src/SFA.DAS.Commitments.Api/SFA.DAS.Commitments.Api.csproj

```diff
@@ -110,6 +110,12 @@
       <HintPath>..\packages\Microsoft.Data.Services.Client.5.7.0\lib\net40\Microsoft.Data.Services.Client.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.IdentityModel.Clients.ActiveDirectory, Version=3.13.8.999, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
ActiveDirectory, ActiveDirectory.Platform & SFA.DAS.Http have been added. Are they the 'entourage' of one of the updated Api.Types?
It doesn't look like they would be needed otherwise.

Reply:
Good spot, I've found the cause and pushed a new commit. These 2 packages are then no longer needed. I also had forgot to downgrade one events API client.